server/static/css/main.css

```diff
@@ -17,3 +17,12 @@ pre code  {
 .no-dash {
   border-bottom: none !important;
 }
+
+.top-align {
+    margin-top: -1em;
+}
+
+td.slim-cell {
```

Comment:
I don't think this is used anymore in this PR - but I can imagine it being useful later.